pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -96,7 +96,7 @@ func (c *openShiftClusterConverter) ToExternal(oc *api.OpenShiftCluster) interfa
 		out.Properties.RegistryProfiles = make([]RegistryProfile, len(oc.Properties.RegistryProfiles))
 		for i, v := range oc.Properties.RegistryProfiles {
 			out.Properties.RegistryProfiles[i].Name = v.Name
-			out.Properties.RegistryProfiles[i].UserName = v.Username
+			out.Properties.RegistryProfiles[i].Username = v.Username
```

Comment:
@mjudeikis I'm a bit out of the loop here - can you explain how simply changing `UserName` to `Username` and `userName` to `username` has the effect of "re-add[ing] the registry profile to the API" and "ignor[ing] conversion from external to internal"?